### DISCUSSION TITLE
Importing old projects into new projects

### DISCUSSION THREAD

**Discussion by nickserv** [community/community#6372] [Projects and Issues] [answered]
I have old project boards with hundreds of issues, pull requests, notes, and lists of those. I would like to use the new projects features as I do prefer its UI, but it would take way too much work to manually recreate all my existing boards. It would be great to have a way to import an old project into a new project with a board view, and have the option of adding additional views.

It seems like I'm also [not able to use the API because of the draft items](https://github.com/github/feedback/discussions/6371).

**Comment by treboryx** [comment]
I also would find this extremely helpful, to be able to import repistory old projects and issues into the new GitHub issues.

_this isn't supposed to be an answer not sure why it's marked as one_

**Comment by ghost** [comment]
yeah, and would also help to map labels to new project fields, e.g. I use labels for type and priority, for example. But just the option to import "en masse" would be great to start with!

**Comment by nickserv** [comment, marked as answer]
This is now possible: [Migrating from projects (classic)
](https://docs.github.com/en/issues/planning-and-tracking-with-projects/creating-projects/migrating-from-projects-classic)
